community/modules/compute/schedmd-slurm-gcp-v5-node-group/variables.tf

```diff
@@ -0,0 +1,334 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# Most variables have been sourced and modified from the SchedMD/slurm-gcp
+# github repository: https://github.com/SchedMD/slurm-gcp/tree/v5.1.0
+
+variable "project_id" {
+  description = "Project in which the HPC deployment will be created."
+  type        = string
+}
+
+## Node Group Definition
+
+variable "name" {
+  description = "Name of the node group."
+  type        = string
+  default     = "ghpc"
+
+  validation {
+    condition     = can(regex("^[a-z](?:[a-z0-9]{0,5})$", var.name))
+    error_message = "Node group name (var.name) must begin with a letter, be fully alphanumeric and be 6 characters or less. Regexp: '^[a-z](?:[a-z0-9]{0,5})$'."
+  }
+}
+
+variable "node_conf" {
+  description = "Map of Slurm node line configuration."
+  type        = map(any)
+  default     = {}
+}
+
+variable "node_count_dynamic_max" {
+  description = "Maximum number of nodes allowed in this partition."
```

Comment:
I might add the word "dynamic" in the description. The way it reads now is not clear how it interacts with static nodes. AKA, do static nodes count toward this max.

Reply:
Fixed